python/ray/rllib/agents/impala/multi_vtrace.py

```diff
@@ -129,6 +129,7 @@ def from_logits(behaviour_policy, target_policy, actions,
             clip_rho_threshold=clip_rho_threshold,
             clip_pg_rho_threshold=clip_pg_rho_threshold)
 
+        # noinspection PyProtectedMember
```

Comment:
Let's remove these everywhere (shouldn't be a part of the pull request as they don't use them)